server/server.js

```diff
@@ -24,12 +26,22 @@ app.use(cors());
 
 app.get('/api/photos', (req, res) => {
   let id = req.query.product_id;
-  console.log('prod id --->', req.query.product_id);
+  // console.log('prod id --->', req.query.product_id);
   db.getAllPhotos(1, (err, result) => {
     res.json(result);
   });
 });
 
+app.post('/api/photos', (req, res) => {
+  for(let i = 0; i < 8; i++) {
+    let oneSeed = seed.seeds[i];
+    db.savePhotos(oneSeed);
+  }
+
```

Comment:
Should the post request be able to save new photos that are attached to the response body?

Reply:
yes you're right actually, good catch! will fix and update

Reply:
@mflinn92